### ISSUE TITLE
Request mode is "same-origin" but the URL's origin is not same as the request origin

### ISSUE THREAD

**Issue by Jaid** [3F/GhrMeter.user.js#1]
Thanks for releasing that userscript, looks great! I can't get it to work though.

Console error:

```
Fetch API cannot load https://api.github.com/repos/Jaid/array-to-object-keys/releases.
Request mode is "same-origin" but the URL's origin is not same as the request origin https://github.com.
```

Screenshot of the exception break in Developer Tools:

![image](https://user-images.githubusercontent.com/6216144/52530904-760e8a80-2d0d-11e9-937f-800772978d70.png)

Screenshot of the security settings in my userscript manager:

![image](https://user-images.githubusercontent.com/6216144/52531176-ac4e0900-2d11-11e9-86a9-2f0108811399.png)

Environment:

```
Linux 4.15.0-44-generic
Linux Mint 19.1 Cinnamon
Google Chrome 72.0.3626.96 (Official Build) (64-bit)
Tampermonkey 4.7.63
```

Am I doing something wrong?

**Comment by 3F** [comment]
Thanks for the report,

This error looks like because of default Tampermonkey features with cross-origin requests. 
The same error for added Tampermonkey 4.8.5847 to Firefox 65.0 (I'm using this). But no problems when Greasemonkey 4.7.

But anyway, yes **api**.github.com it's really other domain to github.com for cors policy, so I changed mode to this.

Please check this out before release:

https://github.com/3F/GhrMeter.user.js/releases/tag/CI-build-20

**Comment by Jaid** [comment]
Thank you very much! Works fine with that patch. :)

![image](https://user-images.githubusercontent.com/6216144/52534459-f64ee300-2d41-11e9-9057-0f4cb539e0b7.png)

**Comment by 3F** [comment]
Just released 0.6.1. Enjoy.

```
* FIXED: Possible "Request mode is "same-origin" but the URL's origin is not same as the request origin" #1
* NOTE: Tested via Firefox 65.0 + Greasemonkey 4.7
* NOTE: Tested via Firefox 65.0 + Tampermonkey 4.8.5847
* NOTE: Tested via Chrome 72.0.3626.96 + Tampermonkey 4.7.63 ( Thanks @Jaid )
```
